gym/envs/toy_text/kellycoinflip.py

```diff
@@ -93,7 +93,7 @@ def __init__(self, initialWealth=25, edgePriorAlpha=7, edgePriorBeta=3, maxWealt
         self.maxRounds = maxRounds
         self.rounds = self.maxRounds
         self.maxWealth = maxWealth
-        self._seed()
+        if reseed: self._seed()
```

Comment:
Won't this crash in `step` if `_seed` isn't called in `__init__`? Otherwise, `self.np_random` is undefined.